components/textbox/src/_mixin.scss

```diff
@@ -271,18 +271,33 @@ $utrecht-support-prince-xml: false !default;
   }
 }
 
+@mixin utrecht-textbox--aantalmarkten-size {
+  /* Nummer onder 9999 */
+  --utrecht-textbox-value-max-length: 10;
+}
+
 @mixin utrecht-textbox--postal-code-nl-size {
   /* Dutch postal code: 4 numbers, 1 space and 2 letters */
   --utrecht-textbox-value-max-length: 7;
 }
 
+@mixin utrecht-textbox--postbus-nummer-size {
+  /* A postbusnummer consists of at least one and at most five digits. */
+  --utrecht-textbox-value-max-length: 2;
+}
+
+@mixin utrecht-textbox--postcode-size {
+  /* A postcode consists of four numbers and two letters. */
+  --utrecht-textbox-value-max-length: 2;
+}
+
 @mixin utrecht-textbox--house-number-size {
   /* Dutch house number is 99999 at most: 5 numbers */
-  --utrecht-textbox-value-max-length: 5;
+  --utrecht-textbox-value-max-length: 1;
 }
 
 @mixin utrecht-textbox--house-letter-size {
-  /* Dutch house letter is 1 letter */
+  /* Dutch house letter can be 1 or 2 letters */
   --utrecht-textbox-value-max-length: 1;
```

Comment:
Uit mijn eigen onderzoek bleek ooit dat het maximaal 1 letter kan zijn. Ik ben benieuwd waaruit blijkt dat het nu uit 2 letters kan bestaan?

Reply:
Ik heb hier weer onderzoek naar gedaan en ontdekte dat ik informatie over huisnummers uit Amerika had bekeken. Inderdaad, in Nederland is het maximaal 1 letter. My bad